scripts/prepublish.js

```diff
@@ -0,0 +1,13 @@
+var fs = require('fs');
```

Comment:
I stuck with `var` so that this still supports 0.10 (bit of an LTS policy wonk) but moved to `const` where appropriate. Happy to migrate to `let` if you'd prefer.